packages/enzyme-adapter-utils/src/Utils.js

```diff
@@ -94,6 +97,10 @@ export function displayNameOfNode(node) {
 
   if (!type) return null;
 
+  if (type === Portal) {
+    return 'Portal';
```

Comment:
Same question as on the other PR; should we add this here, or only on the adapters?